man/org.freedesktop.LogControl1.xml

```diff
@@ -116,6 +116,21 @@ node /org/freedesktop/LogControl1 {
     for details about <varname>BusName=</varname>.)</para>
   </refsect1>
 
+  <refsect1>
+    <title>Example</title>
+
+    <example>
+      <title>Create a simple listener on the bus that implements LogControl1</title>
+
+      <programlisting><xi:include href="logcontrol-example.c" parse="text"/></programlisting>
+
+      <para>This creates a simple client on the bus (the user bus, when run as normal user). It implements
```

Comment:
This implies the system bus when run as root. But I don't think that's the case. `sd_bus_default()` checks if the slice of the caller is called `user-*.slice`. `user@0.service` runs in `/user.slice/user-0.slice`, so you'd get the user bus too.

The docs for `sd_bus_default()` are pretty bad. I couldn't answer this question based on the man page, so I think we should fix the man page.